R/board_connect.R

```diff
@@ -102,28 +102,35 @@ board_rsconnect <- function(auth = c("auto", "manual", "envvar", "rsconnect"),
       if (length(fs::dir_ls(cache)) == 0) {
         # We've never successfully connected
         abort(c(
-          glue("Failed to connect to RSC instance at <{server$url}>"),
+          glue("Failed to connect to Posit Connect instance at <{server$url}>"),
```

Comment:
Is "Failed to connect to Posit Connect" an awkward thing to say? Or fine?

Reply:
I took a quick look at connect synonyms and didn't see anything obviously better.